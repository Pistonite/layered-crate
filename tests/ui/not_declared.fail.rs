use layered_crate::layers;

#[layers]
mod src {
    #[depends_on(x)]
    pub extern crate w;
    pub extern crate x;
}

fn main() {
    w::do_something();
    x::do_something();
}
