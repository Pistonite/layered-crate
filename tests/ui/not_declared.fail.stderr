error[E0432]: unresolved import `crate_::y`
 --> tests/ui/src/w.rs
  |
  | use crate_::y; // doesn't depend on it
  |     ^^^^^^^^-
  |     |       |
  |     |       help: a similar name exists in the module: `x`
  |     no `y` in `w::crate_`

warning: unused import: `x`
 --> tests/ui/not_declared.fail.rs:5:18
  |
5 |     #[depends_on(x)]
  |                  ^
  |
  = note: `#[warn(unused_imports)]` on by default
